README.md

```diff
@@ -649,3 +649,98 @@ Now you should also see the *RRbot* represented correctly in `RViz`.
    ```
 
 3. You should also see the *RRbot* moving in `RViz`.
+
+
+### Scenario showcase: Using ros2_control within a local namespace
+
+- Launch file: [rrbot_namespace.launch.py](ros2_control_demo_bringup/launch/rrbot_namespace.launch.py)
+- URDF: [rrbot.urdf.xacro](ros2_control_demo_bringup/config/rrbot.yaml)
+- ros2_control URDF: [rrbot.ros2_control.xacro](ros2_control_demo_description/rrbot_description/ros2_control/rrbot.ros2_control.xacro)
+- Controllers config: [rrbot_namespace_controllers.yaml](ros2_control_demo_bringup/config/rrbot_namespace_controllers.yaml)
+
+**NOTE:**when running `ros2 control` CLI commands you have to use additional parameter with exact controller manager node name, i.e., `-c /rrbot/controller_manager`.
+
+- Command interfaces:
+  - joint1/position
+  - joint2/position
+- State interfaces:
+  - joint1/position
+  - joint2/position
+
+Available controllers: (nodes under namespace "/rrbot")
+- `forward_position_controller[forward_command_controller/ForwardCommandController]`
+- `joint_state_broadcaster[joint_state_broadcaster/JointStateBroadcaster]`
+
+List controllers:
+```
+ros2 control list_controllers -c /rrbot/controller_manager
+```
+
+Switch controller to use `position_trajectory_controller` (`JointTrajectoryController`):
```

Comment:
Organization thought: It might be worth moving this below the forward position controller command. I was just entering commands in order which didn't work since I switch to the JTC before using the forward_position_controller. The pro to the way it is now is it requires a closer read, the pro to swapping is reducing confusion for new folks.